### PR TITLE
Issue 80

### PR DESCRIPTION
**Pull request by TheHappyDev**
#80 

[issue-80](https://github.com/oddbit/flutter_facebook_app_events/issues/80)
#Starting with iOS 14, you will need to set the setAdvertiserTrackingEnabled flag.
This allows you to inform Facebook whether to use the data to deliver personalized ads in line with your own legal obligations, platform terms, and commitments you’ve made to your users. If the flag is set to false we will not be able to deliver personalized ads.

change allows you to set the field

The first commit is to implement this feature
the second commit is what I needed to do to make the example project work on my local with the latest flutter. 
Feel free to cherry-pick commits and make amendments.

### CODE REVIEW

**Comment by TheHappyDev** [comment]
Any chance this could get reviews and merged in?

File: android/src/main/res/drawable/launch_background.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@android:color/white" />
+
+    <!-- You can insert your own image assets here -->
+    <!-- <item>
+        <bitmap
+            android:gravity="center"
+            android:src="@mipmap/launch_image" />
+    </item> -->
+</layer-list>
```
Comment (DennisAlund) [inline android/src/main/res/drawable/launch_background.xml:12]: This seems like an accidentally added file. To be removed.

File: lib/facebook_app_events.dart [hunk 2]
```diff
@@ -262,6 +262,19 @@ class FacebookAppEvents {
     );
   }
 
+   /// Sets the Advert Tracking propeety for iOS advert tracking 
+   /// an iOS 14+ feature, android should just return a success. 
+  Future<void> setAdvertiserTracking({
+    bool enabled,
```
Comment (DennisAlund) [inline lib/facebook_app_events.dart:268]: Make the parameter `@required`

File: pubspec.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 name: facebook_app_events
 description: Flutter plugin for Facebook App Events, an app measurement
   solution that provides insight on app usage and user engagement in Facebook Analytics.
-version: 0.9.0
+version: 0.10.0
```
Comment (DennisAlund) [inline pubspec.yaml:4]: Don't bump the version in the PR since it will be coordinated with if there are other PRs to include in the same release.

**Review by DennisAlund — COMMENTED** [review]
Thank you for a great PR.

There are just some small feedback, such as cleaning up unrelated resource files in the android folder: `android/src/main/res/...`

**Comment by hurrba** [comment]
any chance this can be merged ?

**Comment by TheHappyDev** [comment]
I have fixed recommendations on the PR

**Comment by DennisAlund** [comment]
> There are just some small feedback, such as cleaning up unrelated resource files in the android folder: android/src/main/res/...

There are still files in the resource folder, as mentioned above. I'll fix them in a branch instead of ping-ponging further. Thank you for your efforts and willingness to support maintaining 🙏

**Pull request merged**